todo/src/components/Todos.js

```diff
@@ -0,0 +1,90 @@
+import React from 'react';
+import { connect } from 'react-redux';
+
+import { addTodo, toggleTodo, deleteTodo, clearCompleted } from '../actions';
+
+import Todo from './Todo';
+
+class Todos extends React.Component {
+    state = {
+        todoValue: '',
+        filterMenu: false,
+        colorMenu: false,
+        todos: []
+    };
+    addTodo = e => {
+        e.preventDefault();
+        if(this.state.todoValue !== '') {
+            this.props.addTodo(this.state.todoValue);
+            this.setState({ todoValue: '' });
+        }
+    }
+    handleChanges = e => this.setState({ todoValue: e.target.value });
+
+    toggleButtons = type => {
+        if(type === 'filter'){
+            this.setState({ filterMenu: !this.state.filterMenu });
+        } else if(type === 'color') {
+            this.setState({ colorMenu: !this.state.colorMenu });
+        }
+    }
+    // onClickHandler = (value) => {
+    //     console.log(value);
+    // }
+   
+    render(props) {
+        return (
+            <div className='todos'>
+                <header className={`header-${this.props.color}`}>
```

Comment:
great use of props to conditionally pass a class properties, some people go out of their way by using a module called classnames to do this but I think this way is better and gives you more control over what you want your component to do.